pyproject.toml

```diff
@@ -48,7 +48,23 @@ path = "pyccel/version.py"
 artifacts = [
   "*.pyccel",
 ]
+include = [
+  "include",
+   "*.py",
+  "*.pyh",
+  "*.pyccel",
+  "pyccel/stdlib/*/*.h",
+  "pyccel/stdlib/*/*.f90",
+  "*.tx",
+  "*.c"
+]
+exclude = [
+    "misc",
+    "src"
+]
```

Comment:
Can you please explain how you have determined what to list under `include` and `exclude`?

This is not totally clear to me. For example, I expected that we should write "pyccel/extensions/STC/misc" instead of just "misc"...

Reply:
@EmilyBourne Do you happen to know the answer?

Reply:
I'm not sure but I can take a guess. The `exclude` folder also doesn't list everything in STC (e.g. `docs` is not listed) but seems to be sufficient to exclude these folders.

I think that once an `include` section is specified it is used to define all files that should be included using this as a regex. Given that the regex for c files is simply `*.c` this is probably pulling in `misc` (pyccel/extensions/STC/misc/)  which contains `*.c` files and `src` (pyccel/extensions/STC/src/) which contains both `*.c` and  `*.py` files.

I agree that it would be better to use more explicit paths for this though as it should avoid several of these conflicts. Do you want me to fix it?

Reply:
I have run a couple of tests and this explanation seems to line up with what I am seeing so the fix is easy.

Reply:
96221ceeb

Reply:
OK, yes. Please modify this file to use more explicit paths.